src/common/components/hosts/MassChangeHostnameModal.tsx

```diff
@@ -291,7 +312,7 @@ const MassChangeHostnameModal: React.FC<MassChangeHostnameModalProps> = ({
             formikActions.setStatus({
               error: {
                 title: t('ai:Failed to update host'),
-                message: getErrorMessage(e),
+                message: isApiError(e) ? e?.response?.data?.reason : getErrorMessage(e),
```

Comment:
Could we use `getApiErrorMessage` which does pretty much the same?
(Unless it doesn't return the proper message if `error.response?.data?.message` is set)

Reply:
Good idea, I missed that function.